### PR TITLE
fix: set correct text for Enter key

### PR DESCRIPTION
**Pull request by jrandolf**

### CODE REVIEW

File: src/bidiMapper/domains/input/ActionDispatcher.ts [hunk 1]
```diff
@@ -567,7 +567,12 @@ export class ActionDispatcher {
     // --- Platform-specific code begins here ---
     // The spread is a little hack so JS gives us an array of unicode characters
     // to measure.
-    let unmodifiedText = [...key].length === 1 ? key : undefined;
+    let unmodifiedText: string | undefined;
```
Comment (OrKoN) [inline src/bidiMapper/domains/input/ActionDispatcher.ts:570]: can we extract it into a helper?

Comment (thiagowfx) [reply src/bidiMapper/domains/input/ActionDispatcher.ts:570]: +1

File: src/bidiMapper/domains/input/ActionDispatcher.ts [hunk 2]
```diff
@@ -640,3 +634,93 @@ export class ActionDispatcher {
     // --- Platform-specific code ends here ---
   }
 }
+
+const getKeyEventText = (key: string, code: string, source: KeySource) => {
+  switch (key) {
+    case 'Enter':
+      return '\r';
+  }
```
Comment (thiagowfx) [inline src/bidiMapper/domains/input/ActionDispatcher.ts:642]: `if (key === 'Enter')`

**Review by thiagowfx — APPROVED** [review]

File: src/bidiMapper/domains/input/ActionDispatcher.ts [hunk 3]
```diff
@@ -640,3 +636,96 @@ export class ActionDispatcher {
...
+        break;
+      case 'Minus':
+        return '\x1F';
+    }
+    return '';
+  }
+  if (source.alt) {
+    return '';
+  }
+  return;
```
[the remark this answers is not in the document]
Comment (jrandolf) [reply src/bidiMapper/domains/input/ActionDispatcher.ts:730]: It's necessary because all paths won't have a return otherwise.

**Pull request merged**